infra/feast-operator/config/crd/bases/feast.dev_featurestores.yaml

```diff
@@ -297,6 +297,9 @@ spec:
                               the DB store persistence for the offline store service
                             properties:
                               secretKeyName:
+                                description: |-
+                                  By default, the DB parameters should be placed as-is from the feature_store.yaml under the `type` key, if
```

Comment:
does `should be placed as-is from the feature_store.yaml under the type key` explain that the secret values must be contained in a secret key with the name same to the value of the `type` field? as it is now, I would be tempted to use a key called `type`, instead.

Reply:
Can you propose of an alternative phrase ?

Reply:
How does this sound ?
`// By default, the DB parameters secret values should be placed as-is from the feature_store.yaml in a secret key with the name same to the value of the type field, if
	// SecretKeyName is specified the DB parameters secret values should be placed as-is from the feature_store.yaml under the specified key`

Reply:
i think its still a little muddy. 
maybe something like this to start... `By default, the selected store "type" is used as the SecretKeyName and should ...`
also, instead of `DB parameters secret values`, maybe `data store parameters` is sufficient?

those might help, but it could be simpler and more clear overall.

Reply:
So is this enough ? @tchughesiv @dmartinol 
`By default, the selected store "type" is used as the SecretKeyName, data store parameters should be placed as-is from the feature_store.yaml`

Reply:
@tmihalac the `as-is` bit worries me... what if they include `type` or `registry_type`? won't that throw an error?

Reply:
Yes it will throw an error

Reply:
@tmihalac is that documented anywhere? maybe this needs to be split up?

`// By default, the selected store "type" is used as the SecretKeyName` stays put over `SecretKeyName`.

Something like this might go over the `SecretRef` field? `// Data store parameters should be placed as-is from the "feature_store.yaml" under the secret key. "registry_type" & "type" fields should be removed.`

Reply:
Sounds good to me

Reply:
confirmed

Reply:
Pushed changes